newbank/server/Customer.java

```diff
@@ -42,6 +43,20 @@ public Boolean passwordMatches(String password) {
 		return this.password.equals(password);
 	}
 
+	
+	public void setMemorableWord(String memorableWord) {
+		// don't allow password resets at the moment
```

Comment:
I would remove this comment (because we are allowing password resets now), or alter it to say something like "don't allow the memorable word to be changed".